docs/pages/distribution/publishing-websites.mdx

```diff
@@ -86,12 +86,23 @@ Learn more about [Netlify](https://www.netlify.com/).
 
 ### Manual deployment with the Netlify CDN
 
-```sh
-npm install netlify-cli -g
-netlify deploy
-```
+1. Install the Netlify CLI by running `npm install netlify-cli -g`
+
+2. Build your Expo web app with `npx expo export:web` (SDK 45 and lower should use `expo build:web` via the legacy global Expo CLI).
```

Comment:
This step can be removed since it's repeating the set of instructions already covered [Creating a build section](https://docs.expo.dev/distribution/publishing-websites/#creating-a-build).